components/dashboard/table/Table.js

```diff
@@ -0,0 +1,31 @@
+import Row from './Row';
+import Pagination from './Pagination';
+import Filter from './Filter';
+
+export default function Table({ title, data, paginationOptions, filters }) {
+  return (
+    <div className=" bg-white lg:w-1/2 border border-gray-200">
+      <div className="inline-flex bg-gray-300 w-full p-4 gap-1">
+        <h2 className="text-2xl font-semibold leading-tight">{title}</h2>
+      </div>
+      {filters && <Filter data={filters} />}
```

Comment:
Please give the filters area some padding around and between them (not too much, just so it breathes a little bit).